tools/ci/cibuild.sh

```diff
@@ -278,7 +278,10 @@ function python-tools {
   export PIP_USER=yes
   export PYTHONUSERBASE=${tools}/pylocal
   add_path "${PYTHONUSERBASE}"/bin
-  pip3 install \
+
+  # Force the reinstall of python packages due to issues with GitHub
+  # cache restoration.
+  pip3 install --force-reinstall \
```

Comment:
Do we need the same for `imgtool` installation?

Reply:
Yes, thanks for the reminder. Actually, there is no need for this special treatment to `imgtool`, I'll merge it to the above list.